popup/popup.js

```diff
@@ -141,7 +141,14 @@ function initPopup() {
           $('label', info).textContent = t('unreachableAMO');
           const note = (FIREFOX < 59 ? t('unreachableAMOHintOldFF') : t('unreachableAMOHint')) +
                        (FIREFOX < 60 ? '' : '\n' + t('unreachableAMOHintNewFF'));
-          const renderToken = s => s[0] === '<' ? $create('b', tWordBreak(s.slice(1, -1))) : s;
+          const renderToken = s => s[0] === '<'
+            ? $create('strong', {
+              textContent: s.slice(1, -1),
+              onclick: handleEvent.copyContent,
+              tabIndex: -1,
```

Comment:
Why `-1`?

Reply:
Initially I was focusing the element to select it's content. Using -1 allows the focus, but you can't tab to it. Since there is an alternative, we don't need that `tabIndex` anymore.